packages/hide-production-sourcemaps/README.md

```diff
@@ -0,0 +1,14 @@
+# zodern:hide-production-sourcemaps
```

Comment:
In the past, it was decided that if Meteor generates production source maps by default, it should also hide them by default. Instead of copying this package over, it would probably be better to make this the default behavior in the `webapp` package, with a way to disable it.

Reply:
Good insight!